pkgs/os-specific/linux/wireguard/default.nix

```diff
@@ -3,12 +3,12 @@
 let
   name = "wireguard-${version}";
 
-  version = "20160708";
+  version = "unstable-2016-07-22";
```

Comment:
thanks for taking into account the new guidelines. Note that, even though it won't make a difference here, `unstable` is supposed to be appended to the name part, and not prepended to the version part.